workers/unity/Assets/Fps/Scripts/UI/FpsUIButton.cs

```diff
@@ -0,0 +1,140 @@
+using UnityEngine;
+using UnityEngine.EventSystems;
+using UnityEngine.UI;
+
+namespace Fps
+{
+    public class FpsUIButton : Button
+    {
+        public Image TargetFrame;
```

Comment:
These should not be public

Reply:
they need to public due to the `FpsUIButtonEditor`